modules/apps/message-boards/message-boards-api/src/main/java/com/liferay/message/boards/service/MBCategoryLocalService.java

```diff
@@ -140,6 +141,9 @@ public void deleteCategory(
 			MBCategory category, boolean includeTrashedEntries)
 		throws PortalException;
 
+	public void deleteCategory(String externalReferenceCode, long groupId)
```

Comment:
We can  have issues with the name of this because we are naming it in other apis as "deleteXXXByExternalReferenceCode"

Reply:
You are right. I've changed the method name in [c93ba2b](https://github.com/liferay-content-management/liferay-portal/pull/5866/commits/c93ba2b03f5b1691a477cbb80a34b128ea5fcb3f)